documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -189,6 +196,12 @@ You can migrate virtual machines to {virt} from the command line.
 *  You must ensure that all xref:prerequisites[prerequisites] are met.
```

Comment:
the comment above also applies to the vddk part in this section
and the note about the fingerprint should also appear in the UI section until https://issues.redhat.com/browse/MTV-530 is resolved

Reply:
I am adding the note to the Create Providers UI PR.